lib/collection/src/operations/types.rs

```diff
@@ -388,11 +388,39 @@ impl Default for ScrollRequestInternal {
     }
 }
 
+fn points_example() -> Vec<api::rest::Record> {
+    let mut payload_map_1 = Map::new();
+    payload_map_1.insert("city".to_string(), Value::String("London".to_string()));
+    payload_map_1.insert("color".to_string(), Value::String("green".to_string()));
+
+    let mut payload_map_2 = Map::new();
+    payload_map_2.insert("city".to_string(), Value::String("Paris".to_string()));
+    payload_map_2.insert("color".to_string(), Value::String("red".to_string()));
+
+    vec![
+        api::rest::Record {
+            id: PointIdType::NumId(43),
```

Comment:
Default example value for `PointIdType` is 42, so it results in a response example where `next_page_offset` is lower than Ids in the `points` (43 and 44). However, this is not possible, scroll API should always return IDs lower than `next_page_offset`. Propose to replace with `40` and `41`